data/events.yaml

```diff
@@ -1,3 +1,31 @@
+- title: Interactive Theorem Proving 2023
+  url: https://mizar.uwb.edu.pl/ITP2023/
+  start_date: July 31 2023
+  end_date: August 4 2023
+  location: Bialystok, Poland
+  type: conference
+
+- title: Conference on Intelligent Computer Mathematics 2023
+  url: https://cicm-conference.org/2023/cicm.php
+  start_date: September 4 2023
+  end_date: September 8 2023
+  location: Cambridge, UK
+  type: conference
+
+- title: Certified Programs and Proofs 2023
+  url: https://popl23.sigplan.org/home/CPP-2023
+  start_date: January 16 2023
+  end_date: January 17 2023
+  location: Boston, MA, USA
+  type: conference
+
+- title: Certified Programs and Proofs 2024
+  url: https://popl24.sigplan.org/home/CPP-2024
+  start_date: January 15 2024
+  end_date: January 16 2024
+  location: London, UK
+  type: conference
```

Comment:
Are these intended to be in chronological order?

Reply:
They get sorted when the page is generated, and the yaml list is out of order anyway, so I didn't bother!